manifests/init.pp

```diff
@@ -405,6 +405,14 @@
   }
 
   if $configure_repo {
+    # Advise the user that codership and percona do not have a EL 8 repos
```

Comment:
I'm undecided on this, these things get dated pretty quickly. I'd rather see this mentioned in the README under "Limitations".